src/input.ts

```diff
@@ -60,15 +60,15 @@ export function getActionInput(): Input {
     user: getInput("user"),
     organization: getInput("organization"),
     owner: getInput("owner") || context.repo.owner,
-    repo: getInput("repo") || context.repo.repo,
+    rateLimit: getBooleanInput("rate-limit"),
   }
 }
 
-export function isRestPackageType(type?: PackageType): type is PackageType.Container | PackageType.Npm {
-  return type !== undefined && [PackageType.Container, PackageType.Npm].includes(type)
-}
-
 export function validateInput(input: Input): Input {
+  if (!input.type) {
+    throw new Error("package feed type is required so must be supplied")
```

Comment:
Since we already validate in `getTypeInput´, this is not needed I think.

Reply:
Yep - should be good